pom.xml

```diff
@@ -67,7 +67,7 @@
         <git-plugin.version>3.1.0</git-plugin.version>
         <workflow-support-plugin.version>2.14</workflow-support-plugin.version>
         <scm-api-plugin.version>2.0.8</scm-api-plugin.version>
-        <groovy-cps.version>1.18</groovy-cps.version>
+        <groovy-cps.version>1.19-20170815.195847-1</groovy-cps.version> <!-- TODO: Update to release once https://github.com/cloudbees/groovy-cps/pull/69 is merged/released -->
```

Comment:
AFAICT this is unnecessary: no `src/main/` change is being done there. So this does not need to be downstream of https://github.com/cloudbees/groovy-cps/pull/69 at all. Which simplifies your life because this can be merged without cutting a `groovy-cps` release.